.github/workflows/ci.yml

```diff
@@ -14,5 +14,8 @@ jobs:
         with:
           node-version: 20.x
           cache: pnpm
-      - run: pnpm install
-      - run: pnpm run fmt:global-check
+      - run: |
+          pnpm install
+          pnpm run fmt:global-check
+          # -i is to ignore packages that are expected to have multiple versions across the workspace
+          npx sherif@latest -i @openzeppelin/contracts
```

Comment:
I'm not a fan of this - it means that CI can break from one run to another because of new `sherif` rules.

Why not add it as a dependency to the root of the repository?

Reply:
This is their recommended way of running it: https://github.com/QuiiBz/sherif

I agree using `latest` is a mistake, I'll pin it to a specific version.

>  It can even be part of the fmt:global-check

It can be, but I am unsure if it's related to formatting 🤷

Reply:
Pinned the version sir

Reply:
I migrated it to the package.json file, and one thing that I missed straight away is the comments 🙈 I have no way to explain what the `-i` flag does and why certain packages are excluded

Reply:
`git blame` :stuck_out_tongue:

True that JSON doesn't allow for comments which is annoying :disappointed: